whiteBoarding/splitMiddleStr/keg-splitMiddleStr.js

```diff
@@ -0,0 +1,13 @@
+// Here's a start...10-28-18
+
+str1 = abcdefghijklmnopqrstuvxyz
+str2 = 123456789
+
+function(str1, str2){
```

Comment:
Syntax error when run in `node`
```/Users/edwardweymouth/code/practice/keg-middle.js:4
 function(str1, str2){
         ^

SyntaxError: Unexpected token (```

Why?

Reply:
b/c I didn't give the function a name.